mesos-client/src/main/scala/com/mesosphere/mesos/client/CredentialsProvider.scala

```diff
@@ -60,15 +60,16 @@ case class DcosServiceAccountProvider(uid: String, privateKey: String, root: URL
     */
   private def claim: JsObject = Json.obj("uid" -> uid, "exp" -> expireIn(DcosServiceAccountProvider.CLAIM_LIFETIME))
```

Comment:
`DcosServiceAccountProvider.CLAIM_LIFETIME` is probably defined far away from this line. The name is a bit cryptic. This defines the lifetime of the service login token. Can we rename that variable, or maybe just write an explicit `20` here in the code so that it's also clear that this needs to be kept in sync with the code comment `@return a claim for the user that expires in twenty seconds.`?

Just a nit of course .. :-)